release-notes/testing.yml

```
@@ -1,4 +1,8 @@
 releases:
+  42.20250512.2.0:
+    issues:
+      - text: "Zincati still apply an update even when rolled back"
```

I updated the title of the issue because I thought this sounded better:

```suggestion
      - text: "Zincati still applies an update even when rolled back"
```